CuriousKatie/Helper/Helper.swift

```diff
@@ -0,0 +1,145 @@
+        let interest32 = Interest(name: "swimming", category: Category.sport, gear: [Gear.swimwear], environment: Environment.anywhere)
+        let interest33 = Interest(name: "traveling", category: Category.social, gear: [Gear.backpack, Gear.photo], environment: Environment.outdoor)
+        let interest34 = Interest(name: "watching movies", category: Category.social, gear: [Gear.noGear], environment: Environment.indoor)
+        let interest35 = Interest(name: "writing", category: Category.art, gear: [Gear.noGear], environment: Environment.anywhere)
+        let interest36 = Interest(name: "yoga", category: Category.sport, gear: [Gear.sportClothes], environment: Environment.anywhere)
+        
+        return [interest01,
+                interest02,
+                interest03,
+                interest04,
+                interest05,
+                interest06,
+                interest07,
+                interest08,
+                interest09,
+                interest10,
+                interest11,
+                interest12,
+                interest13,
+                interest14,
+                interest15,
+                interest16,
+                interest17,
+                interest18,
+                interest19,
+                interest20,
+                interest21,
+                interest22,
+                interest23,
+                interest24,
+                interest25,
+                interest26,
+                interest27,
+                interest28,
+                interest29,
+                interest30,
+                interest31,
+                interest32,
+                interest33,
+                interest34,
+                interest35,
+                interest36]
+    }
+    
+    /// Generate all the people
+    static func generateAllPeople() -> [Person] {
+        let person01 = Person()
+        let person02 = Person()
+        let person03 = Person()
+        let person04 = Person()
+        let person05 = Person()
+        let person06 = Person()
+        let person07 = Person()
+        let person08 = Person()
+        let person09 = Person()
+        let person10 = Person()
+        let person11 = Person()
+        let person12 = Person()
+        
+        return [person01,
+                person02,
+                person03,
+                person04,
+                person05,
+                person06,
+                person07,
+                person08,
+                person09,
+                person10,
+                person11,
+                person12]
+    }
+    
+    /// Shuffle interests and pick a random amount between 1 and the maximum number of interests per person
+    /// *Note: in case the number of interests is smaller than maximum number of interests per person return at least the available;e number of interests
+    static func pickSomeInterests() -> [Interest] {
+        let pool = generateAllInterests().shuffled()
```

Comment:
You can have a slight performance gain by only generating the interests once.

Reply:
Here I'm trying to pick a random number and random interests for each person. I didn't have any other idea on how to do it 😐

Reply:
Fixed 😎